src/views/Info/Tokens/TokenPage.tsx

```diff
@@ -72,7 +72,11 @@ const TokenPage: React.FC<RouteComponentProps<{ address: string }>> = ({
 
   // Needed to scroll up if user comes to this page by clicking on entry in the table
   useEffect(() => {
-    window.scrollTo(0, 0)
+    window.scrollTo({
```

Comment:
Nice. Could you do me a favor by creating a common hook `useScrollTop`  in the `hooks` folder ? It would avoid copy and paste in the future.